sites/public/lib/hooks.ts

```diff
@@ -30,3 +32,37 @@ export const useFormConductor = (stepName: string) => {
   }, [conductor])
   return context
 }
+
+const listingsFetcher = function (listingsService: ListingsService) {
+  return (_url: string, page: number, limit: number) => {
+    const params = {
+      page,
+      limit,
+    }
+    return listingsService?.list(params)
+  }
+}
+
+// TODO: move this so it can be shared with the partner site.
+export function useListingsData(pageIndex: number, limit = 10) {
+  const { listingsService } = useContext(AuthContext)
+  const { data, error } = useSWR(
+    [`${process.env.backendApiBase}/listings`, pageIndex, limit],
```

Comment:
There's a separate env variable for this - I think it's `listingServiceUrl`, see the old `getStaticProps`.

Reply:
Updated